tools/phpbench/composer.lock

```diff
@@ -0,0 +1,1750 @@
+{
```

Comment:
Yes, the `.lock` file is necessary for reproducible results here. In my local environment, I also froze the CPU clock to ensure results are consistent with each other.